staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -363,6 +371,18 @@ func (s *Scheme) AddFieldLabelConversionFunc(version, kind string, conversionFun
 	return nil
 }
 
+// AddDownwardAPIFieldLabelConversionFunc adds a conversion function to convert
+// field label (a reference to a field in an object in Downward API) of the
+// given kind from the given version to internal version representation.
+func (s *Scheme) AddDownwardAPIFieldLabelConversionFunc(version, kind string, conversionFunc FieldLabelConversionFunc) error {
```

Comment:
scheme is focused on things that apply to all objects... does downward API apply to anything other than pods?

Reply:
It's only for pods. Do we have a better place for these functions?

Reply:
How about treating this conversion function as a utility function, and put it in `pkg/apis/core/pods/...`. The [Scheme.ConvertDownwardAPIFieldLabel](https://github.com/kubernetes/kubernetes/pull/55902/files#diff-ee0234a823e9084b1155334e646f42e2R509) method doesn't provide any additional value anyway. The only users are kubelet and validation.